src/Http/Http.Extensions/test/RequestDelegateFactoryTests.cs

```diff
@@ -4251,6 +4251,73 @@ string HelloName(string name)
         Assert.Equal(400, httpContext.Response.StatusCode);
     }
 
+    [Fact]
+    public async Task RequestDelegateFactory_InvokesFilters_OnDelegateWithTarget()
+    {
+        // Arrange
+        var httpContext = CreateHttpContext();
+        httpContext.Request.Query = new QueryCollection(new Dictionary<string, StringValues>
+        {
+            ["name"] = "TestName"
+        });
+
+        // Act
+        var factoryResult = RequestDelegateFactory.Create((string name) => "Hello, {name}!", new RequestDelegateFactoryOptions()
+        {
+            RouteHandlerFilterFactories = new List<Func<RouteHandlerContext, RouteHandlerFilterDelegate, RouteHandlerFilterDelegate>>()
+            {
+                (routeHandlerContext, next) => async (context) =>
+                {
+                    Console.WriteLine(context.Parameters[0]);
```

Comment:
Console.WriteLine is odd in a test 😃

Reply:
Yeah, not totally necessary for this test. Will nuke it.

Reply:
😢